cql3/statements/select_statement.hh

```diff
@@ -107,7 +108,7 @@ protected:
     bool _range_scan_no_bypass_cache = false;
     std::unique_ptr<cql3::attributes> _attrs;
 protected :
-    virtual future<::shared_ptr<cql_transport::messages::result_message>> do_execute(service::storage_proxy& proxy,
+    virtual future<::shared_ptr<cql_transport::messages::result_message>> do_execute(service::storage_proxy& proxy, service::forward_service& forwarder,
         service::query_state& state, const query_options& options) const;
```

Comment:
Wasn't there a previous patch that added this parameter?

Reply:
It's not really relevant anymore due to `do_execute` signature change.